components/function-controller/internal/controllers/serverless/system_state.go

```diff
@@ -244,6 +251,13 @@ func (s *systemState) buildJob(configMapName string, cfg cfg) batchv1.Job {
 	if s.instance.Spec.RuntimeImageOverride != "" {
 		args = append(args, fmt.Sprintf("--build-arg=base_image=%s", s.instance.Spec.RuntimeImageOverride))
 	}
+
+	var resourceRequirements corev1.ResourceRequirements
```

Comment:
We can extract to to seperate method to avoud repetition.

Reply:
Done